examples/lua_REPL/Makefile

```diff
@@ -18,12 +18,12 @@ BOARD_INSUFFICIENT_MEMORY := bluepill calliope-mini cc2650-launchpad \
                              b-l072z-lrwan1  cc2538dk ek-lm4f120xl  feather-m0 \
                              ikea-tradfri limifrog-v1 mbed_lpc1768  nrf6310 \
                              nucleo-f091rc  nucleo-l073rz  nz32-sc151 \
-                             openmote-cc2538 openmote-b  pba-d-01-kw2x \
+                             openmote-b openmote-cc2538 pba-d-01-kw2x \
```

Comment:
These changes are unrelated. If you want to sort them, please put that in a separate PR.

Reply:
@miri64 Back to original order, thanks!